src/Model.elm

```diff
@@ -269,5 +287,13 @@ update msg model =
                 Err error ->
                     { model | publicTimeline = [], errors = (errorText error) :: model.errors } ! []
 
+        UserAccount result ->
+            case result of
+                Ok account ->
+                    { model | account = Just account } ! []
+
+                Err error ->
+                    { model | account = Nothing, errors = (errorText error) :: model.errors } ! []
```

Comment:
I feel bad about us keeping duplicating similar logic for these `case..of` statements. Could you see any way to factor them?